node/overseer/overseer-gen/tests/ui/gen.rs

```diff
@@ -0,0 +1,80 @@
+pub struct Ooooh<Ctx, Sub0, Sub1> {
+	sub0: Sub0,
+	yyy: Sub1,
+	spawner: S,
+	metrics: Metrics,
+}
+
+impl<Ctx, Sub0, Sub1> Ooooh<Ctx, Sub0, Sub1>
+where
+	Sub0: Subsystem<Ctx>,
+	Sub1: Subsystem<Ctx>,
+{
+	fn builder() -> OooohBuilder {
+		OooohBuilder::default()
+	}
+}
+
+struct OooohBuilder<Ctx, Sub0, Sub1> {
+	sub0: ::std::option::Option<Sub0>,
+	yyy: ::std::option::Option<Sub1>,
+	spawner: ::std::option::Option<spawner>,
+	metrics: ::std::option::Option<metrics>,
```

Comment:
prefix `::` is not needed

Reply:
Technically it is, `std` could refer to a in-scope module and hence the wrong `Option` type could be used.